src/Footer.js

```diff
@@ -0,0 +1,17 @@
+const location = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789",
+];
+
+export function Footer() {
+  return (
+    <div className="footer">
+      <ul>
+        {location.map((element, index) => (
+          <li key={index}>{element}</li>
```

Comment:
Happy to see you have the `key` attribute for each `li`

reference of key https://legacy.reactjs.org/docs/lists-and-keys.html#keys